src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -252,19 +255,21 @@ using BloomFilter = boost::dynamic_bitset<>;
 // it where needed. Having a Bloom filter that always returns true avoids a branch in the hot loop.
 static const auto ALL_TRUE_BLOOM_FILTER = ~BloomFilter(BLOOM_FILTER_SIZE);
 
-// @param in_table             Table to materialize
-// @param column_id            Column within that table to materialize
-// @param histograms           Out: If radix_bits > 0, contains one histogram per chunk where each histogram contains
-//                             1 << radix_bits slots
-// @param radix_bits           Number of radix_bits, needed only for histogram calculation
-// @param output_bloom_filter  Out: A filled BloomFilter where `value & BLOOM_FILTER_MASK == true` for each value
-//                             encountered in the input column
-// @param input_bloom_filter   Optional: Materialization is skipped for each value where the corresponding slot in the
-//                             Bloom filter is false
+// @param in_table                Table to materialize
+// @param column_id               Column within that table to materialize
+// @param histograms              Out: If radix_bits > 0, contains one histogram per chunk where each histogram
+//                                contains 1 << radix_bits slots
+// @param radix_bits              Number of radix_bits, needed only for histogram calculation
+// @param output_bloom_filter     Out: A filled BloomFilter where `value & BLOOM_FILTER_MASK == true` for each value
+//                                encountered in the input column
+// @param partition_node_location Out: Contains the NUMA node location at position j of the returned partition at
+//                                position j.
+// @param input_bloom_filter      Optional: Materialization is skipped for each value where the corresponding slot
+//                                in the Bloom filter is false
```

Comment:
Ah, I see, there are a handful of out-parameters already. Hm, then I guess you adding another is somewhat fair.